### DISCUSSION TITLE
Getting "GET http://localhost:3000/api/auth/me 400 Bad request" error in console.

### DISCUSSION THREAD

**Discussion by AmeerMuaviya** [nextauthjs/next-auth#7937] [Help]
### Question 💬

Getting `GET http://localhost:3000/api/auth/me 400 Bad request` error in console. I never called this endpoint in my entire application. My app and next-auth is working fine but this error is annoying me

### How to reproduce ☕️

I implemented **Next-auth** in my application.
This is my `/api/auth/[...nextauth]/route.js` file
``` js
import NextAuth from "next-auth";
import GoogleProvider from "next-auth/providers/google";
import Auth0Provider from "next-auth/providers/auth0";
import GithubProvider from "next-auth/providers/github";

const handler = NextAuth({
  providers: [
    GoogleProvider({
      clientId: process.env.GOOGLE_CLIENT_ID,
      clientSecret: process.env.GOOGLE_CLIENT_SECRET,
    }),
    Auth0Provider({
      clientId: process.env.AUTH0_CLIENT_ID,
      clientSecret: process.env.AUTH0_CLIENT_SECRET,
      issuer: process.env.AUTH0_ISSUER,
    }),
    GithubProvider({
      clientId: process.env.GITHUB_ID,
      clientSecret: process.env.GITHUB_SECRET,
    }),
  ],
});
export { handler as GET, handler as POST };
```
i'm simply getting user like this
 ``` js 
import { useSession, signIn, signOut } from "next-auth/react"
```
# here is the screenshot of the problem
![image](https://github.com/nextauthjs/next-auth/assets/88961101/15ebfeb0-e2fa-464e-848f-2d971a9d4e7d)

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by shasherazi** [comment]
Did anyone find a solution to this?

**Comment by VirajVaitha123** [comment]
Hey,

This is years later, but hopefully the solution helps others.

The problem for me, was that I was using Auth0 and switched to NextAuth. Inside the providers page, I had included the older provider too.

For example:
```
export const Providers: FCWC = ({ children }) => {
  return (
    <AuthProvider>
    <UserProvider>
      <FilesProvider>
          <ThemeProvider defaultTheme="dark" enableSystem={false} attribute="class">
            <QueryClientProvider {...{ client }}>
              <ReactQueryDevtools />
              {children}
            </QueryClientProvider>
          </ThemeProvider>
      </FilesProvider>
      </UserProvider>
      </AuthProvider>
  );
};
```
Remove </UserProvider> and the problem should be fixed.

Hope this helps.
